whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/OrgSpec.groovy

```diff
@@ -3039,16 +3039,12 @@ class OrgSpec extends BaseQueryUpdateSpec {
 
         then:
         def ack = ackFor message
-        ack.failed
```

Comment:
Please add new tests to add, modify, remove "country:" code on an organisation *with* resources maintained by the RIPE NCC, this should still not be allowed.

Reply:
Added 3 more tests